### PR TITLE
Make captions bold text below the plots

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**

### CODE REVIEW

**Comment by Vaibhavdixit02** [comment]
plot_17
plot_41
plot_47
plot_49
plot_13

**Comment by ChrisRackauckas** [comment]
That's not what is meant by captions. They should not be in the plot but in the text. It should be like:

**Morris global sensitivity results.** Depicted are the results of a Morris global sensitivity analysis using the `morris` function of GlobalSensitivity.jl with parameters x y z. The plots are results of the plot recipe depicting the mean and variance of the Morris indices over the parameter space x in a to b etc.

One line is not a figure caption in real work. It should be like 4 sentences minimum.

File: joss/paper.md [hunk 1]
```diff
@@ -86,39 +86,37 @@ ax, hm = CairoMakie.heatmap(fig[2,1], reg_sens.standard_regression,
...
         color = :red, axis = (xticksvisible = false,
         xticklabelsvisible = false, title = "Predator (Morris)",))
+Label(fig[2, :], "Morris method GSA for Lotka Volterra model")
 fig
 ```
 
-#### Morris method GSA for Lotka Volterra model
-
-![morrisscat](https://user-images.githubusercontent.com/23134958/127019346-2b5548c5-f4ec-4547-9f8f-af3e4b4c317c.png)
+![Scatter plot of `μ*` for the mean of the prey population and maximum of the predator population from the Morris method. The first plot corresponds to the mean of the prey population and the second plot corresponds to the maximum of the predator population. Each of the four parameters occur in both plots as one scatter point with `μ*` on the y-axis and index of the parameter corresponding to its order on the x-axis.](https://user-images.githubusercontent.com/23134958/177251158-69a6e8b8-0a46-4998-b9f2-106886827176.png)
```
Comment (ChrisRackauckas) [inline joss/paper.md:112]: First, second, etc. doesn't have a clear meaning. "Left: the Morris indices for the mean of the prey population". Top left, bottom right, etc. If you really took the time you'd add (A) etc. to the plots for naming, but left right top bottom should be fine.

Comment (ChrisRackauckas) [inline joss/paper.md:112]:
```suggestion
![**Scatter plot of `μ*` for the mean of the prey population and maximum of the predator population from the Morris method.** Depicted are the Morris indices for the mean of the time series solutions for the Lotka-Volterra ODE solved on the time interval `(0.0,10.0)` with parameters as demonstrated in the code sample. Each of the four parameters occur in both plots as one scatter point with `μ*` on the y-axis and index of the parameter corresponding to its order on the x-axis. Left: the Morris indices for the mean of the prey population. Right: the Morris indices for the maximum of the predator population.](https://user-images.githubusercontent.com/23134958/177251158-69a6e8b8-0a46-4998-b9f2-106886827176.png)
```

Comment (ChrisRackauckas) [reply joss/paper.md:112]: A similar treatment needs to be done with the other captions, and then I think we're good to resubmit.

**Comment by ChrisRackauckas** [comment]
I think we're good.

**Pull request merged**